src/app/navbar/navbar.component.html

```diff
@@ -103,11 +76,14 @@
       <a mat-button class="icon icon-sm" href="http://docs.dockstore.org/">Docs</a>
     </div>
   </mat-toolbar-row>
-  <mat-toolbar-row class="visible-xs toolbar-holder text-center toolbar-holder-sm">
-    <div class="container-fluid navbar-holder">
-      <span *ngIf="isLoggedIn" fxLayout="row">
-        <ng-container [ngTemplateOutlet]="loggedIn"></ng-container>
-      </span>
-    </div>
-  </mat-toolbar-row>
 </mat-toolbar>
+<!-- This is the login button used in both navbar sizes -->
+<ng-template #loginButton>
+  <button data-cy="dropdown-main" *ngIf="isLoggedIn" mat-raised-button color="accent" id="dropdown-main" [matMenuTriggerFor]="menuLarge">
+    <mat-icon class="hidden-sm hidden-md hidden-lg" *ngIf="user?.isAdmin">lock</mat-icon>
```

Comment:
Is this saying that admins get a lock icon, curators get an edit icon, and anyone else gets a circle? 
I think we might need a follow-up ticket for more intuitive icons.